tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py27,py34
```

Comment:
did python3 support break, or are you just removing this because we don't officially support python3 currently?